.github/workflows/build.yml

```diff
@@ -117,22 +113,41 @@ jobs:
       with:
         name: report-dapr-java-sdk-actors
         path: sdk-actors/target/jacoco-report/
-    - name: Get pom parent version
-      run: |
-        PARENT_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
-        echo "PARENT_VERSION=$PARENT_VERSION" >> $GITHUB_ENV
-    - name: Is SNAPSHOT release ?
-      if: contains(github.ref, 'master') && contains(env.PARENT_VERSION, '-SNAPSHOT')
-      run: |
-        echo "DEPLOY_OSSRH=true" >> $GITHUB_ENV
-    - name: Is Release or RC version ?
-      if: startswith(github.ref, 'refs/tags/v') && !contains(env.PARENT_VERSION, '-SNAPSHOT')
-      run: |
-        echo "DEPLOY_OSSRH=true" >> $GITHUB_ENV
-    - name: Publish to ossrh
-      if: env.DEPLOY_OSSRH == 'true' && env.JDK_VER == '11'
-      run: |
-        echo ${{ secrets.GPG_PRIVATE_KEY }} | base64 -d > private-key.gpg
-        export GPG_TTY=$(tty)
-        gpg --batch --import private-key.gpg
-        mvn -V -B -Dgpg.skip=false -s settings.xml deploy -pl \!examples
+  publish:
+    runs-on: ubuntu-latest
+    needs: build
+    env:
+      JDK_VER: 11
```

Comment:
The previous set of steps is run off the JDK matrix right? Do we care/need to duplicate that here?